src/tests/Register.test.js

```diff
@@ -64,4 +65,68 @@ it('allows the user to register successfully', async () => {
 
     expect(screen.getByLabelText('response')).toHaveTextContent("User was created successfully. A confirmation email has been sent via email. After confirming your email you can login.")
     
-})
\ No newline at end of file
+})
+it('Check empty field warning', async () => {
+        
+    const { getByLabelText }= render(<Register />)
+   
+    
+     fireEvent.change(screen.getByLabelText("Name :",{selector :"input"} ), {
+            target: { value: '' },
+        })
+    
+
+     fireEvent.change(screen.getByLabelText("Username :",{selector :"input"} ), {
+            target: { value: '' },
+        })
+ 
+     fireEvent.change(screen.getByLabelText("Email :",{selector :"input"} ), {
+           target: { value: '' },
+         })
+     
+     fireEvent.change(screen.getByLabelText("Password :",{selector: "input"}), {
+          target: { value: '' },
+         }) 
+    
+        act(() => {
+            fireEvent.click(screen.getByRole('button', { name: "Sign Up" }), {
+                target: { value: 'true' },
+            })
+        });
+        expect(getByLabelText("Name :",{selector : "input"})).toBeRequired();
+        expect(getByLabelText("Username :",{selector : "input"})).toBeRequired();
+        expect(getByLabelText("Email :",{selector : "input"})).toBeRequired();
+        expect(getByLabelText("Password :",{selector : "input"})).toBeRequired();
+    }) 
+    it('Validation message warning', async () => {
```

Comment:
Be careful of indentation. This is a new test so it doesn't need the indentation.